shared/src/main/kotlin/org/javacs/kt/classpath/Home.kt

```diff
@@ -1,18 +1,28 @@
 package org.javacs.kt.classpath
 
-import org.javacs.kt.util.userHome
+import java.nio.file.Files
+import java.nio.file.Path
 import java.nio.file.Paths
+import org.javacs.kt.util.KotlinLSException
+import org.javacs.kt.util.userHome
 
+private fun createPathOrNull(envVar: String): Path? {
+  return System.getenv(envVar)?.let { Paths.get(it) }
+}
```

Comment:
```suggestion
private fun createPathOrNull(envVar: String): Path? = System.getenv(envVar)?.let(Paths::get)
```

Reply:
btw, please configure your editor to use 4 spaces too

Reply:
Ah sorry my work enforces 2 spaces. But I'll keep that in mind for my changes here/the future.